Compile ug-minor-fixes branch on ML cluster and hook mozilla repos for all submodules

abhi-agg opened the pull request:
- Added .compute file with the compilation instructions
 - Hooked mozilla repos for all of its submodules (non-recursively):
      examples, regression-tests, nccl, sentencepiece, fbgemm and
      simple-websocket-server
 - Compiles successfully on ML cluster

abhi-agg committed:
Compile ug-minor-fixes branch on ML cluster

 - Added .compute file with the instructions

abhi-agg committed:
Hook mozilla repo for 'examples' submodule

kdavis-mozilla reviewed (changes requested):
Why are there changes in the examples directory?

abhi-agg committed:
Hook mozilla repo for 'regression-tests' submodule

kdavis-mozilla commented:
Oh you're moving all the submodules over, thus the changes in examples, regression-tests...

abhi-agg commented:
> Oh you're moving all the submodules over, thus the changes in examples, regression-tests...

Yes, I thought the change related to hooking mozilla repos for submodules also belonged in the same commit.

abhi-agg committed:
Hook mozilla repo for 'nccl' submodule

abhi-agg committed:
Hook mozilla repo for 'sentencepiece' submodule

abhi-agg committed:
Hook mozilla repo for 'fbgemm' submodule

abhi-agg committed:
Hook mozilla repo for 'simple-websocket-server' submodule

abhi-agg committed:
Fix submodule 'nccl' to 2.3.7-1

 - commit b56650c7f59b8cd40d18809784a6d6be38ef8acb
   is used to compile successfully

kdavis-mozilla commented on .compute:
```diff
@@ -0,0 +1,44 @@
+    unzip \
+    wget
+
+# Build the server 
```
I don't think this comment is correct.

COMPILE_SERVER defaults to OFF and it's not set to ON here.

abhi-agg replied:
Agreed. Changed it in the next commit.

kdavis-mozilla reviewed (commented):
Just one "nit". The comment in the .compute seems to be wrong. Or am I wrong?

abhi-agg committed:
Changed comment in .compute file

kdavis-mozilla reviewed (approved):
LGTM
